app/invocation/invocation.tsx

```diff
@@ -115,6 +115,10 @@ export default class InvocationComponent extends React.Component<Props, State> {
     }, 3000);
   }
 
+  decodedSearchParam(key: string) {
```

Comment:
`urlSearchParams.get()` should already return the decoded value -- I think the problem is that we are double-encoding. In `InvocationFilterComponent.handleFilterChange` we're encoding `event.target.value` and then passing that to `router.updateParams`, which then calls `getModifiedUrl` which encodes the value again.

These days we should be able to avoid ever calling `encode`/`decodeURIComponent` and instead use `URLSearchParams` for everything (it automatically encodes/decodes for us).

Reply:
Good catch, fixed!